### DISCUSSION TITLE
Showing the website domain

### DISCUSSION THREAD

**Discussion by MrMohamedAbdallah** [dailydotdev/daily#596] [Feature Requests]
Now that what we see, The blog card has the logo of the website
![image](https://user-images.githubusercontent.com/51543900/175825032-d5c8ad2f-3434-4c73-aaf5-b28868781e3c.png)

In order to know the website that we're going to be redirected to we need to hover on the icon. And we can't know the domain of it.
![image](https://user-images.githubusercontent.com/51543900/175825081-9a824e04-c664-4dcf-bf2a-dd91ede928e9.png)

**Wanted Result**: The root domain should be showmen without hovering and when we hover on the root domain we should see the full domain.

**Example**
![image](https://user-images.githubusercontent.com/51543900/175825260-e0b63360-b3cc-4361-a44e-0d463bc3f067.png)

**Hover state**
![image](https://user-images.githubusercontent.com/51543900/175825470-601e7ba1-0fcb-4a83-8b66-a425a457be1b.png)

**Comment by idoshamun** [comment]
Hi @MrMohamedAbdallah,

Why would you want to see the root domain of the redirected website?

**Reply by MrMohamedAbdallah** [reply]
Hi @idoshamun,
Some titles are really interested, But once I open the article and I see the one who posted it I know it just a spam/drama/misleading etc. And at the time I don't want to block that user.

And vise-versa, sometimes the title isn't that interested but when I know the blog who posted it I know it's going to be informative.

PS: by "spam/drama/misleading" I mean according to my preferences.

**Reply by idoshamun** [reply]
I see now. Thanks for providing extra details. We will take it into consideration
